docs/adr/0016-publishing-times-and-change-history.md

```diff
@@ -0,0 +1,214 @@
+new revisions for superseded editions - this feels logically incorrect as these
+superseded editions would no longer accurately represent the data they
+published to GOV.UK.
+
+### Backdating content
+
+The backdating feature in Content Publisher supports only backdating the first
+edition of a piece of content and not subsequent ones as is allowed by
+Whitehall. This is because the Publishing API does not have the means to change
+the initial item in the change history for content and thus would cause a
+change history that is inconsistent with the `first_published_at` time. This
+has a consequence for publishers that if they make a mistake with backdating
+or forget to backdate in Content Publisher then they cannot rectify this
+without creating a new document or requesting support from developers.
+
+To resolve these problems we considered 3 approaches:
+
+### Correct Whitehall data in the Publishing API
+
+We considered doing an audit and syncing exercise for times and change
+histories from Whitehall publisher to the Publishing API. This would
+involve either doing a large data modification exercise or creating new endpoints
+in the Publishing API that allow importing this data.
+
+The positive of this approach was that it could mean Content Publisher
+continues relying on the Publishing API for time and change history management
+without needing this data stored in Content Publisher.
+
+The negatives to this was that there would be no guarantees this data would
+remain accurate once the work was complete, nor would there be an easy way to
+check whether times are accurate when migrating Whitehall data to Content
+Publisher. This also didn't allow any improvements in removing/editing change
+history or allowing backdating past the first edition.
+
+### Store times and change history in Content Publishing and sync them to Publishing API
+
+We considered changing Content Publisher to be similar in approach to
+Whitehall and be the canonical source of publishing times and change history.
+Each time Content Publisher content is updated the Publishing API would be
+sent the times and change history.
+
+This approach offered was a greater degree of control over data, which allows
+Content Publisher to enhance the backdating feature and provide more support for
+change history removal/editing. This also offers a higher degree of
+compatibility with Whitehall for migrating content due to it being a similar
+model.
+
+This offered a negative that it increased Content Publisher's responsibilities
+to storing time and change history. It also means that prior to publishing
+content the Publishing API would need times updating which adds an additional
+API call during publishing.
+
+### Adapt Publishing API to support an altering change history
+
+We considered modifying the Publishing API so that it could provide
+an API means to allow the altering of change histories. This would involve
```

Comment:
Maybe `... could provide an API to allow ...`.  `means` doesn't really add anything here.